netty/src/test/java/io/grpc/transport/netty/NettyClientHandlerTest.java

```diff
@@ -184,14 +183,13 @@ public void cancelBufferedStreamShouldChangeClientStreamStatus() throws Exceptio
 
     assertTrue(createPromise.isSuccess());
     verify(stream).transportReportStatus(eq(Status.CANCELLED), eq(true),
-        any(Metadata.Trailers.class));
+            any(Metadata.Trailers.class));
```

Comment:
No .... it seems that the IDE's config for whitespace in this case is imperfect.  It's probably more effort than it's worth to revert and there aren't may occurrences of this.